apps/address-book/contracts/AddressBook.sol

```diff
@@ -142,16 +142,16 @@ contract AddressBook is AragonApp {
     /**
      * @notice Get index associated to entry `_addr` from the registry.
      * @dev getter for the entries mapping for an index in entryArr
-     * @param _addr The ID of the entry to get
+     * @param _addr The Ethereum address of the entry to get
      * @return contentId pointing to the IPFS structured content object for the entry
      */
     function getEntryIndex(address _addr) external view isInitialized entryExists(_addr) returns (uint256 index) {
         index = entries[_addr].index;
     }
 
     /**
-     * @notice Checks if an entry exists in the registry
-     * @param _entry the address to check
+     * @notice Checks if an `_entry` exists in the registry
```

Comment:
should probably remove the "an" here, but there is no situation where this radspec will be evaluated, so that's kind of just semantics